services/brig/src/Brig/Team/API.hs

```diff
@@ -142,6 +142,18 @@ routesPublic = do
     Doc.response 200 "Invitation successful." Doc.end
     Doc.errorResponse invalidInvitationCode
 
+  -- FUTUREWORK: Add another endpoint to allow resending of invitation codes
+  head "/teams/invitations/by-email" (continue headInvitationsByEmailH) $
```

Comment:
Is this a good name for an endpoint? Anyone has suggestions for improvements?

Reply:
it's perfect!  :)